classpath/java/lang/System.java

```diff
@@ -114,9 +110,11 @@ private static Properties makeProperties() {
   public static native long currentTimeMillis();
 
   public static native int identityHashCode(Object o);
+  
+  private static native long getBaseNanoTime();
```

Comment:
This is the new native call.  I could have just made the nanoTime() call the native function, but it works either way.